rmw_connextdds_common/src/ndds/dds_api_ndds.cpp

```diff
@@ -212,21 +212,56 @@ rmw_connextdds_initialize_participant_qos_impl(
   return RMW_RET_OK;
 }
 
+static
+rmw_ret_t
+initialize_cft_parameters(
```

Comment:
for consistency with other symbols, please add prefix `rmw_connextdds_` to function name.

Reply:
updated